src/app/(BottomTabNavigation)/Cases/CaseScreen/EligibilityForm/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import styles from './styles';
+import { updateCaseStatus } from '../../../../../supabase/queries/cases';
+import { CaseUid, Eligibility } from '../../../../../types/types';
+import Arrow from '../../../../../../assets/arrow.svg';
+
+export default function EligibilityForm() {
+  const router = useRouter();
+  const { caseId } = useLocalSearchParams<{
```

Comment:
Let's use `const { caseId } = useLocalSearchParams() as unknown as { caseId: CaseUid };` this instead for consistency in how we use local params. We need to make sure we do error handling from the screen we are passing this down from to do this though.